server/src/routes/clientRouter.js

```diff
@@ -3,7 +3,7 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new", "/movies", "/movies/:id"];
+const clientRoutes = ["/", "/user-sessions/new", "/users/new", "/movies", "/movies/:id", "/movies/:id/new-review"];
```

Comment:
this route does not exist in React - your new review form is on the same page as the movie?